src/packages.builds

```diff
@@ -12,7 +12,7 @@
     <PackagingTaskDir Condition="'$(BuildToolsTargets45)' == 'true'">$(BinDir)AnyOS.AnyCPU.$(ConfigurationGroup)/Microsoft.DotNet.Build.Tasks.Packaging.Desktop/</PackagingTaskDir>
   </PropertyGroup>
 
-  <Import Project="Microsoft.DotNet.Build.Tasks\PackageFiles\packages.targets" Condition="'$(ImportGetNuGetPackageVersions)' != 'false'" />
+  <Import Project="$(ToolsDir)packages.targets" Condition="'$(ImportGetNuGetPackageVersions)' != 'false'" />
```

Comment:
I don't think you need this after my change here: https://github.com/dotnet/buildtools/blob/5e01895842ec91bc5b7ba4cd6be296415c6707f7/src/packages.builds#L7